package.json

```diff
@@ -4,14 +4,16 @@
   "version": "4.0.0",
   "author": "Brightcove, Inc.",
   "bin": {
-    "vjsstandard": "./bin/cmd.js"
+    "vjsstandard": "./cli.js"
   },
   "bugs": {
     "url": "https://github.com/videojs/standard/issues"
   },
   "dependencies": {
+    "babel-eslint": "^6.0.0",
```

Comment:
It uses babylon 6.x, but I don't think it's harmful - it's really only there to parse ES6 code and doesn't interact with the rest of our babel 5.x stuff.